mw-webapp/src/component/viewSwitcher/ViewSwitcher.cy.tsx

```diff
@@ -0,0 +1,57 @@
+import Sinon from "cypress/types/sinon";
+import {ViewOption, ViewSwitcher, ViewSwitcherProps} from "src/component/viewSwitcher/ViewSwitcher";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+import {View} from "src/utils/LocalStorageWorker";
+import styles from "src/component/viewSwitcher/ViewSwitcher.module.scss";
+
+const VIEW_SWITCHER_CY = "view-switcher";
+const CARD_ICON_CY = "card-view";
+const TABLE_ICON_CY = "table-view";
+
```

Comment:
Let's use camelCase )

Reply:
Please specify, for variable names or values

Reply:
For values in this case

Reply:
Done